lib/_tls_wrap.js

```diff
@@ -97,9 +97,9 @@ function loadSession(self, hello, cb) {
   }
 
   if (hello.sessionId.length <= 0 ||
-      hello.tlsTicket ||
+      // hello.tlsTicket ||
```

Comment:
This was added in https://github.com/nodejs/node/commit/dda22a520b8f9247938acd9d791d28bc1da8b450 to prevent resumeSession being executed on Session Tickets, but we want that to happen.